infra/Pos/DHT/Real/Real.hs

```diff
@@ -145,12 +147,15 @@ kademliaGetKnownPeers
        , Bi DHTKey
        )
     => KademliaDHTInstance
-    -> m [NetworkAddress]
+    -> m [Peer]
 kademliaGetKnownPeers inst = do
     let kInst = kdiHandle inst
     let initNetAddrs = bool [] (kdiInitialPeers inst) (kdiExplicitInitial inst)
     buckets <- liftIO (K.viewBuckets $ kInst)
-    extendPeers (kdiKey inst) initNetAddrs buckets
+    addresses <- extendPeers (kdiKey inst) initNetAddrs buckets
+    -- TBD kademlia node classification? We'll assume they're all relay nodes.
```

Comment:
What's our direction regarding Kademlia? Will Kademlia peers be allowed to claim their own type? Perhaps we need a new type: `NodeUnclassified`?